tests/src/rules/no-duplicates.js

```diff
@@ -412,6 +411,20 @@ import {x,y} from './foo'
       output: "import Bar, { Foo } from './foo';\nexport const value = {}",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
+
+    /**
+     * Sadly I cannot run this test because I am getting a fatal parsing error:
+     *
+     * `AssertionError [ERR_ASSERTION] [ERR_ASSERTION]: A fatal parsing error occurred: Parsing error: Identifier 'Foo' has already been declared`
+     *
+     * If there is some option to disable this check, we could uncomment this test again.
+     */
+    // #1812 merged imports should not have duplicate import specifiers
+    // test({
+    //   code: "import { Foo, Foo1 } from './foo';\nimport { Foo, Bar } from './foo';",
```

Comment:
indeed, this is invalid code because the `Foo` is duplicated. It wouldn't parse in the first place, so eslint can't fix it.